src/timer.rs

```diff
@@ -23,7 +24,10 @@ impl UserData for Timer {
     fn add_fields<'lua, F: UserDataFields<'lua, Self>>(fields: &mut F) {
         fields.add_field_method_get("limit", |_, this| Ok(this.limit));
         fields.add_field_method_get("time", |_, this| Ok(this.time));
-        fields.add_field_method_set("time", |_, this, val: f64| Ok(this.time = val));
+        fields.add_field_method_set("time", |_, this, val: f64| {
+            this.time = val;
+            Ok(())
+        });
```

Comment:
This was a mistake that `cargo clippy` caught. I was previously returning the result of the expression `this.time = val`, which returns nothing.